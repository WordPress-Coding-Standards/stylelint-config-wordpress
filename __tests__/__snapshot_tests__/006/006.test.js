'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '006', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '006' )],
		configFile: systemTestUtils.caseConfig( '006' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
