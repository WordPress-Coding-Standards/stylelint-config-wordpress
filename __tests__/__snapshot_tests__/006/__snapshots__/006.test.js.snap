// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`006 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "006/stylesheet.scss",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 5,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 13,
        "line": 10,
        "rule": "declaration-colon-newline-after",
        "severity": "error",
        "text": "Expected newline after \\":\\" with a multi-line declaration (declaration-colon-newline-after)",
      },
      Object {
        "column": 13,
        "line": 24,
        "rule": "declaration-colon-newline-after",
        "severity": "error",
        "text": "Expected newline after \\":\\" with a multi-line declaration (declaration-colon-newline-after)",
      },
      Object {
        "column": 23,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 42,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 52,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 71,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 81,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 100,
        "line": 7,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 29,
        "line": 9,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 48,
        "line": 9,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 33,
        "line": 10,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 52,
        "line": 10,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 20,
        "line": 11,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 39,
        "line": 11,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 20,
        "line": 12,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 39,
        "line": 12,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 21,
        "line": 13,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 40,
        "line": 13,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 23,
        "line": 17,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 41,
        "line": 17,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 24,
        "line": 18,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 43,
        "line": 18,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 30,
        "line": 20,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 49,
        "line": 20,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 36,
        "line": 24,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 55,
        "line": 24,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 22,
        "line": 29,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 41,
        "line": 29,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 24,
        "line": 30,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 43,
        "line": 30,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 37,
        "line": 31,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 56,
        "line": 31,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 14,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 45,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 19,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 33,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 22,
        "line": 39,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 40,
        "line": 39,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 24,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 43,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 30,
        "line": 45,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 49,
        "line": 45,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 38,
        "line": 47,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 57,
        "line": 47,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 6,
        "line": 25,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 3 tabs (indentation)",
      },
      Object {
        "column": 4,
        "line": 31,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 2 tabs (indentation)",
      },
      Object {
        "column": 5,
        "line": 47,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 3 tabs (indentation)",
      },
      Object {
        "column": 3,
        "line": 46,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
    ],
  },
]
`;
